workflow-templates/im-build-tf-auto-plan-and-comment-on-prs.yml

```diff
@@ -1,41 +1,47 @@
-# Workflow Code: DeterminedPorcupine_v5    DO NOT REMOVE
+# Workflow Code: DeterminedPorcupine_v6    DO NOT REMOVE
 
 name: Automatically run tf plan and comment on PR
 on:
   pull_request:
     types: [opened, reopened, synchronize]
+    # TODO:  Verify the paths where changes should trigger this workflow.  This can be moved entirely if all changes should trigger the workflow.
```

Comment:
```suggestion
    # TODO:  Verify the paths where changes should trigger this workflow.  This can be removed entirely if all changes should trigger the workflow.
```